### DISCUSSION TITLE
Hooray for Jupyterbook

### DISCUSSION THREAD

**Discussion by pgierz** [executablebooks/meta#757] [Thanks]
Hello,

as the "thanks" part of open source work somehow always falls out the window: thank you for building JupyterBook! It's a fantastic tool, and we are using it to document our Earth System Model. Beware, the handbook is still very very much in it's early stages, but if anyone wants to use it for inspiration: https://awi-esm-handbook.readthedocs.io/en/latest/intro.html

Have a good start to the week!
Paul

**Comment by chrisjsewell** [comment]
Thanks @pgierz that's very kind 🙏
